td-agent/Rakefile

```diff
@@ -710,10 +710,12 @@ class LinuxPackageTask < PackageTask
       spec = YAML.load(`gem specification #{gem_file}`)
       relative_path = gem_file.sub(/#{Dir.pwd}\//, "")
       unless spec.licenses.empty?
+        spdx_compatible_license = spec.licenses.first.sub(/Apache 2.0/, "Apache-2.0")
+                                    .sub(/Apache License Version 2.0/, "Apache-2.0")
```

Comment:
`.` in `/Apache 2.0/` should be escaped.

Reply:
Thanks, orz.
I'll fix it.

Reply:
I've fixed it.